Formula/collada-dom.rb

```diff
@@ -30,19 +25,22 @@ def install
     system "cmake", ".", *std_cmake_args
     system "make", "install"
   end
-end
-
-__END__
-diff --git a/CMakeLists.txt b/CMakeLists.txt
-index 72b6deb..0c7f7ce 100644
---- a/CMakeLists.txt
-+++ b/CMakeLists.txt
-@@ -100,7 +100,7 @@ endif()
 
- if( APPLE OR ${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
-   # apple doesn't have 64bit versions of file opening functions, so add them
--  add_definitions("-Dfopen64=fopen -Dfseeko64=fseeko -Dfseek64=fseek -Dftell64=ftell -Dftello64=ftello")
-+  add_definitions(-Dfopen64=fopen -Dfseeko64=fseeko -Dfseek64=fseek -Dftell64=ftell -Dftello64=ftello)
- endif()
-
- set(COLLADA_DOM_INCLUDE_INSTALL_DIR "include/collada-dom")
+  test do
+    (testpath/"test.cpp").write <<~EOS
+      #include <iostream>
+      #include <dae.h>
+      #include <dae/daeDom.h>
+
+      using namespace std;
+
+      int main()
+      {
+        cout << GetCOLLADA_VERSION() << endl;
+        return 0;
+      }
+    EOS
+    system ENV.cxx, "test.cpp", "-I#{include}/collada-dom2.5", "-L#{lib}", "-lcollada-dom2.5-dp", "-o", "test"
+    assert_equal `./test`.chomp, "1.5.0"
```

Comment:
thanks for the PR! How come this tests true when the version is 2.5.0 and not 1.5.0 ? Assuming this number will change it wouldn't it be better to do something here that wouldn't change over time so it doesn't have to be updated.

Reply:
It does not return the current version, but the DAE file version taken in charge. 
So this test will continue with a future release.

Reply:
@javian Is this answer sufficient? or do you want me to provide another test?

Reply:
Thanks for the explanation @jcarpent, it is a good test.  I have added a comment, and I think we're good to go now.

@javian anything else?

Reply:
Sounds perfect. Thanks for your quick feedbacks!